Demo/Common/Networking/CancelOrderRequest.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+internal struct CancelOrderRequest: Request {
+
+    internal typealias ResponseType = CancelOrderResponse
+
+    internal let order: PartialPaymentOrder
+
+    internal let path = "orders/cancel"
+
+    internal var counter: UInt = 0
+
+    internal var method: HTTPMethod = .post
+
+    internal var headers: [String: String] = [:]
+
+    internal var queryParameters: [URLQueryItem] = []
+
+    internal func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+
+        let configurations = ConfigurationConstants.current
+
+        try container.encode(order, forKey: .order)
+        try container.encode(configurations.merchantAccount, forKey: .merchantAccount)
```

Comment:
probably okay to call `ConfigurationConstants.current.merchantAccount`, since it is used only once

Reply:
too long :-)